effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -385,9 +385,14 @@ object Transformer {
         emit(callLabel(returnAddress, values.map(transform)))
         RetVoid()
 
-      case machine.NewStack(variable, prompt, frame, rest) =>
-        emit(Comment(s"newStack ${variable.name}"))
-        emit(Call(variable.name, Ccc(), transform(variable.tpe), newStack, List(transform(prompt))));
+      case machine.PushNewStack(prompt, frame, rest) =>
```

Comment:
Can we just rename this to `reset`?

Reply:
IMO shift/reset/resume sound too high-level for machine

Reply:
But that's exactly what it is, regardless of the level.